src/Extension/FilterExtension.php

```diff
@@ -252,6 +252,19 @@ public function getFilterData(bool $withPredefined = true)
         return $data;
     }
 
+    /**
+     * @return int
+     */
+    public function getFilterCount()
+    {
+        $count = 0;
+        foreach ($this->getFilterColumns(true) as $columns) {
```

Comment:
Der FilterCount soll ohne vordefinierte Filter sein! --> `$this->getFilterColumns(false)` verwenden.